### PR TITLE
libisofs 1.5.4 (new formula)

### PR DESCRIPTION
**Pull request by cepawiel**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/libisofs.rb [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  url "http://files.libburnia-project.org/releases/libisofs-1.5.4.tar.gz"
+  sha256 "aaa0ed80a7501979316f505b0b017f29cba0ea5463b751143bad2c360215a88e"
+  license "GPL-2.0-or-later"
+
+  depends_on "libtool" => :build
+  depends_on "libzip"
+
+  def install
+    # Fix flat namespace usage.
+    inreplace "configure", "${wl}-flat_namespace ${wl}-undefined ${wl}suppress", "${wl}-undefined ${wl}dynamic_lookup"
```
Comment (SMillerDev) [inline Formula/libisofs.rb:13]: This should be reported upstream and fixed there.

File: Formula/libisofs.rb [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+    system "./configure", *std_configure_args,
+                          "--prefix=#{prefix}"
```
Comment (SMillerDev) [inline Formula/libisofs.rb:16]: Already in the standard arguments
```suggestion
    system "./configure", *std_configure_args
```

File: Formula/libisofs.rb (hunk 1 above)
Comment (cepawiel) [reply Formula/libisofs.rb]: I was unable to open an issue upstream or create a merge request fix as I don't have an account to the projects git server. 

I've added a call to "./bootstrap" before calling configure which should now output the correct flags with the updated changes from libtool without manually patching configure. When updated for a future release it should be possible to remove these extra steps.

**Review by chenrui333 — APPROVED** [review]

**Comment by chenrui333** [comment]
@cepawiel, thanks for your contribution to Homebrew! 🎉 🥇

Without awesome contributors like you, it would be impossible to maintain Homebrew to the high level of quality users have come to expect. Thank you!!!!

**Pull request closed without merging**